app/channels/notifications_channel.rb

```diff
@@ -3,8 +3,8 @@ def subscribed
     stream_for current_user
   end
 
-  def check_unread
-    if current_user.notifications.unread.any?
+  def check_unread(data)
+    if current_user.notifications.unread.where(project_id: data['project_id']).any?
```

Comment:
I think this will result in an error since the notifications table doesn't have a `project_id` column